mmdet/datasets/pipelines/formatting.py

```diff
@@ -226,9 +226,18 @@ def __call__(self, results):
             results = self._add_default_meta_keys(results)
             if len(img.shape) < 3:
                 img = np.expand_dims(img, -1)
-            img = np.ascontiguousarray(img.transpose(2, 0, 1))
+            # To improve the computational speed by by 3-5 times, apply:
+            # If image is not contiguous, use
+            # `numpy.transpose()` followed by `numpy.ascontiguousarray()`
+            # If image is already contiguous, use
+            # `torch.permute()` followed by `torch.contiguous()`
+            if not img.flags.c_contiguous:
```

Comment:
We can also refer to this PR so that the readers can find more details by adding the link in the comments like
```
Refer to https://github.com/open-mmlab/mmdetection/pull/9533 for more details
```

Reply:
Done.